Vic2ToHoI4/Data_Files/blankMod/output/common/on_actions/00_on_actions.txt

```diff
@@ -287,20 +259,20 @@ on_actions = {
 				set_country_flag = achievement_failed_sunset_invasion
 			}
 			# Romania shouldn't declare war on Bulgaria on historical mode, if both are going to join the Axis
-			if = {
-				limit = {
-					original_tag = BUL
-					FROM = { original_tag = GER }
-					is_historical_focus_on = yes
-				}
-				ROM = {
-					add_ai_strategy = {
-						type = befriend
-						id = "BUL"
-						value = 200
-					}
-				}
-			}
+			#if = {
```

Comment:
Good catch, but delete instead of commenting out (feel free to leave a comment explaining why).

Reply:
Done